web/renderer/components/ConnectionsAndDatabases/DatabaseTypeLabel.tsx

```diff
@@ -0,0 +1,39 @@
+import { getDatabaseType } from "@components/DatabaseTypeLabel";
+import { DatabaseConnectionFragment } from "@gen/graphql-types";
+import css from "./index.module.css";
+
+type Props = {
+  conn: DatabaseConnectionFragment;
+};
+
+export function DatabaseTypeLabel({ conn }: Props) {
+  const type = getDatabaseType(conn.type ?? undefined, !!conn.isDolt);
+  switch (type) {
+    case "Dolt":
```

Comment:
Is `type` not an enum?

Reply:
no, it's a string returned from `getDatabaseType`